### ISSUE TITLE
How did you get your inpainting models

### ISSUE THREAD

**Issue by YCC-324** [AdelaideAuto-IDLab/Februus#1]
Hello, I'm studying your paper recently, and I want to know how you get your inpainting models like 'cifar10_inpainting'. I can't find your code for training.
I'd appreciate it if you can provide the code. Thank you

**Comment by baogiadoan** [comment]
hi @YCC-324 , as mentioned in the paper, I adopt the inpainting method from the paper [22] with WGAN-GP loss function, you can have a look at that repo (https://github.com/otenim/GLCIC-PyTorch) while I'm dedicating the time for code releasing.

**Comment by YCC-324** [comment]
@baogiadoan Thank you. It's really nice of you
